transactions.md

```diff
@@ -6,9 +6,61 @@ One transaction event occurs at authorisation time. The other transaction event
 
 ## Transaction Object
 
-##### API version from `2022-07-13`
+| **Field name** | **Description** | **Type** |
+| --- | ------ | --- |
+| `accountId` | The unique identifier of the user account at Fidel API | string |
+| `amount` | The amount of the transaction in the currency it was charged in | number |
+| `approvalCode` | Unique code that is sent along with the authorization of the transaction by Amex. It mirrors the identifiers.amexApprovalCode property. | string (or null) |
```

Comment:
Some sentences end with period (.), others do not. Let's choose one for consistency.

Reply:
Added to all descriptions.